### ISSUE TITLE
Some views are not being automatically deployed

### ISSUE THREAD

**Issue by relud** [mozilla/bigquery-etl#507]
I suspect this applies to views defined in `sql/*/*/view.sql`, but I don't know for sure.

`sql/telemetry/main_summary_v4/view.sql` was modified on Saturday, but I noticed today that the change still hadn't been deployed in `moz-fx-data-derived-datasets`, so I ran it manually. In addition, `moz-fx-data-shared-prod:telemetry.main_summary_v4` didn't even exist until I deployed it.

@jklukas maybe you know what's going wrong here?

cc @sunahsuh because I believe you recently merged a view modification that you didn't manually deploy.

**Comment by jklukas** [comment]
View updates happen as part of schema deploys. Those often happen every day, but looks like we had a gap of ~6 days between end of last week and a deploy 2 days ago:

```
$ gsutil ls gs://moz-fx-data-prod-dataflow/schemas/ | tail -n3
gs://moz-fx-data-prod-dataflow/schemas/201911070121_d2c5dbc.tar.gz
gs://moz-fx-data-prod-dataflow/schemas/201911130131_6a53465.tar.gz
gs://moz-fx-data-prod-dataflow/schemas/201911140118_a4f6df0.tar.gz
```

So I would guess everything is working as expected, and @whd simply hadn't done a prod schema deploy for a while.

**Comment by relud** [comment]
> So I would guess everything is working as expected, and @whd simply hadn't done a prod schema deploy for a while.

okay, but what about the part where the view didn't exist:

> In addition, `moz-fx-data-shared-prod:telemetry.main_summary_v4` didn't even exist until I deployed it.

**Comment by jklukas** [comment]
> okay, but what about the part where the view didn't exist:
> 
> > In addition, `moz-fx-data-shared-prod:telemetry.main_summary_v4` didn't even exist until I deployed it.

Indeed, that's troubling. We should check with @whd about whether logs show anything about this view creation potentially not actually being applied.

**Comment by jklukas** [comment]
Resolved in #610
